### PR TITLE
Pagetree legend

### PR DESCRIPTION
**Pull request by vxsx**
adds pagetree legend 
started restructuring pagetree scss into more maintainable chunks

screen shot 2016-04-21 at 13 38 18

### CODE REVIEW

File: cms/static/cms/sass/components/pagetree/_legend.scss [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    }
+
+    .cms-pagetree-dropdown {
+        display: inline-block;
+    }
+
+    .cms-pagetree-dropdown-menu {
+        right: 100%;
+        left: auto;
+        margin-right: 10px;
```
Comment (lory87) [inline cms/static/cms/sass/components/pagetree/_legend.scss]: ordering

File: cms/static/cms/sass/components/pagetree/_legend.scss [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+            margin-right: 10px;
+        }
+        .cms-icon {
+            margin-right: 8px;
+        }
+    }
+
+    .cms-icon {
+        margin-left: -2px;
+        width: 20px;
```
Comment (lory87) [inline cms/static/cms/sass/components/pagetree/_legend.scss]: here as well

**Comment by lory87** [comment]
overall LGTM

**Comment by czpython** [comment]
👍  Ok to merge

**Pull request merged**